docs/se/health/01_health.adoc

```diff
@@ -139,24 +144,33 @@ NOTE: Health check requires the `JSON-P` support to be enabled. See the example
 
 === Example
 
+[source,java]
+.Enable JSON-P support on the server
```

Comment:
This is also not needed - `HealthSupport` uses the JSON-P writer directly not to interfere with user's application choice of media types.